lru.go

```diff
@@ -436,8 +436,9 @@ func (lru *LRU[K, V]) addWithLifetime(hash uint32, key K, value V,
 	return evicted
 }
 
-// Add adds a key:value to the cache.
-// Returns true, true if key was updated and eviction occurred.
+// Add adds a key:value entry to the cache.
+// The lifetime of the entry is set to the default lifetime.
```

Comment:
Be more precise about the origin of the lifetime and that it is configured on the LRU level.
```suggestion
// Add adds a key:value entry to the cache with the configured lifetime of the LRU.
```